changelogs/fragments/47938-docker_swarm_service-requirements.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
```

Comment:
How about making this a bugfix (i.e. `bugfixes:`)?

Reply:
Just a bugfix or a minor change and a bugfix?

Reply:
I would make it simply a bugfix. And maybe also add that the parameter was simply ignored for that docker-py version before this fix. (I can't really think of any reason why the old behavior would have been useful...)